hack/bin/cabal-run-integration.sh

```diff
@@ -46,7 +46,7 @@ run_integration_tests() {
   service_dir="$TOP_LEVEL/services/$package"
 
   cd "$service_dir"
-  "$TOP_LEVEL/services/integration.sh" \
+  "/usr/bin/python3" "$TOP_LEVEL/services/run-services" \
```

Comment:
Shouldn't this be the Nix's python?

Reply:
Yes, thanks! I forgot we still have to figure out how to make this work in nix.